### DISCUSSION TITLE
Is it possible to deploy Argocd on an Argocd?

### DISCUSSION THREAD

**Discussion by sourceful-karlson** [argoproj/argo-cd#16726] [Q&A] [answered]
I have made an ad hoc install of ArgoCD

Now I would like to Gitops my deployment of ArgoCD on ArgoCD (Ideally, using Helm inside Kustomise)

Is it possible to do that due to the recursive relationship here?

**Comment by andrii-korotkov-verkada** [comment]
Theoretically it should be possible, though I haven't tested it myself. On ArgoCD contributors meeting there was some comment as if doing this deploy is quite normal. I'd explicitly ask on the next contributors meeting.

**Comment by rumstead** [comment, marked as answer]
Sure can! Check out these docs.

https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#manage-argo-cd-using-argo-cd
